tests/devices/test_default_qubit_autograd.py

```diff
@@ -395,7 +395,7 @@ def circuit(x):
         assert all(np.allclose(res[i, :, i], expected[:, i], atol=tol, rtol=0) for i in range(3))
 
     def test_jacobian_agrees_backprop_parameter_shift(self, tol):
-        """Test that jacobian of a QNode with an attached default.qubit.autograd device
+        """Test that jacobian of a QNode with an attached default.mixed.autograd device

```

Comment:
```suggestion
        """Test that jacobian of a QNode with an attached default.qubit.autograd device
```

This seems to be default.qubit, not default.mixed.